src/status_im/ethereum/json_rpc.cljs

```diff
@@ -6,7 +6,15 @@
             [status-im.native-module.core :as status]
             [status-im.utils.money :as money]
             [status-im.utils.types :as types]
-            [taoensso.timbre :as log]))
+            [taoensso.timbre :as log]
+            [cognitect.transit :as transit]))
+
+(def reader (transit/reader :json))
+
+(defn deserialize
```

Comment:
isn't used for now?

Reply:
yeah indeed I was using it at first but it turns out some things don't work anymore